src/App.jsx

```diff
@@ -1,43 +1,27 @@
-import React, {useState, useEffect} from 'react';
+import React from 'react';
 import {BrowserRouter as Router} from 'react-router-dom';
 import './App.css';
-import LoadingScreen from './views/components/LoadingScreen';
```

Comment:
I'm personally inclined to keep the splash screen. It doesn't look like much right now but the original design was one of our better opportunities for us to add Fennel Labs' personality to the app.

Reply:
I like splash screens and this is worthy of more discussion. Right now, it only delays loading the app arbitrarily. This causes me to be inclined to remove it and only add it back in under more consideration, especially with respect to value added to the end user.

Loading time is a an issue that end users care about and it is good to avoid unnecessarily increasing that time. Especially when considering some end users will have varying connection speeds/latency.